pkg/asset/installconfig/openstack/validation/machinepool.go

```diff
@@ -44,6 +44,12 @@ func ValidateMachinePool(p *openstack.MachinePool, ci *CloudInfo, controlPlane b
 		if p.RootVolume.Size < minimumStorage {
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("rootVolume").Child("size"), p.RootVolume.Size, fmt.Sprintf("Volume size must be greater than %d to use root volumes, had %d", minimumStorage, p.RootVolume.Size)))
 		}
+
+		allErrs = append(allErrs, validateZones(p.RootVolume.Zones, ci.VolumeZones, fldPath.Child("rootVolume").Child("zones"))...)
```

Comment:
Should we consider it an error if `len(p.RootVolume.Zones)` is greater than the number of replicas? My guess is we don't really care since it's not affecting how the installer works.

Reply:
it's a good point, but in this case we need to validate compute zones too. with can do it in another pr.